<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Упаковка с оценкой</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_Source">
       <property name="title">
        <string>Исходные фигуры</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="PolygonView" name="graphicsView_Source"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_Result">
       <property name="title">
        <string>Результат</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="PolygonView" name="graphicsView_Result"/>
          <widget class="QTableWidget" name="tableWidget_Result"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Calculate">
      <property name="text">
       <string>Вычислить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PolygonView</class>
   <extends>QGraphicsView</extends>
   <header>polygonview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
